finda-auth/src/main/proto/student.proto

```
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package auth;
+
+import "auth.proto";
```

Buf (1.66.1)

[error] import "auth.proto": file does not exist

(COMPILE)